### DISCUSSION TITLE
All my 8.0 sdk builds broke last night.. How can I determine what the previous docker image was for that tag?

### DISCUSSION THREAD

**Discussion by erichexter** [dotnet/dotnet-docker#5965] [Q&A] [answered]
Trying to create certificates the following fails on the 8.0 sdk build that was published on 10/8/2024. The resulting error is.
 `An error has occurred while exporting the certificate: System.InvalidOperationException: The directory '/sln/artifact' does not exist.  Choose permissions carefully when creating it.`
 
 If I revert to the 8.0.402 container tag, this error goes away.

**Comment by mthalman** [comment]
Since the issue is with the .NET SDK itself, I would recommend logging an issue at https://github.com/dotnet/sdk. Be sure to include repro steps.

**Reply by erichexter** [reply]
I will open a ticket with them for the root cause.  In terms of the docker image tags, Is there some history or log that is publicly visible where I could see what image was used for a tag like 8.0 when it is updated to a new base image?

**Reply by MichaelSimons** [reply, marked as answer]
The dotnet/versions repo has detailed build info for the .NET docker images.  Specifically [this file](https://github.com/dotnet/versions/blob/main/build-info/docker/image-info.dotnet-dotnet-docker-main.json) contains the details about the images built, what tags they have, their layers, the commit sha they were built from etc.  The file is updated with every build.  You can look through the history to find all of the images.

**Reply by MichaelSimons** [reply]
FYI there was an issue logged for what appears to be the same issue - https://github.com/dotnet/sdk/issues/44034
